lightning/src/util/ser.rs

```diff
@@ -802,9 +805,12 @@ impl Readable for Vec<u8> {
 	}
 }
 
-impl_for_vec!(ecdsa::Signature);
-impl_for_vec!(crate::ln::channelmanager::MonitorUpdateCompletionAction);
-impl_for_vec!((A, B), A, B);
+impl_writeable_for_vec!(ecdsa::Signature);
+impl_readable_for_vec!(ecdsa::Signature);
+impl_writeable_for_vec!(crate::ln::channelmanager::MonitorUpdateCompletionAction);
+impl_readable_for_vec!(crate::ln::channelmanager::MonitorUpdateCompletionAction);
+impl_writeable_for_vec!((A, B), A, B);
+impl_readable_for_vec!((A, B), A, B);
```

Comment:
nit: Maybe we could have another convenience macro that does both `impl_writeable_for_vec!` and `impl_readable_for_vec!` for types that don't have a distinction.